.project

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not critical, but this file should be under .gitignore. This is your local setting, which should not be under GIT